src/constants/constants.h

```diff
@@ -8,6 +8,7 @@
 
 // minimum height in cm from the camera's position that an overhead obstacle
 // should be
-#define CLEARANCE_HEIGHT 60
+#define CLEARANCE_HEIGHT 60         // cm
+#define ERR_TOLERANCE_HORZ_GATES 30 // cm
```

Comment:
Hmm, after a bit of thought, I think we should be using `static const` instead of `#define` here. (ie. `static const double RIGHT = -1`). This has several advantages, including being more explicit about the type of `RIGHT` and allowing for proper scoping. For differences between `#define` and `static const` see https://stackoverflow.com/questions/1637332/static-const-vs-define?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa. Putting these in a namespace would also be nice rather then just having them all at global scope, ie:

```
namespace subbots {
    namespace global_constants {
        static const float clearance_height = 60;           // cm
        static const float err_tolerance_horz_gates = 30;   // cm
    }
}
```

Sorry for the late change on this one, I know I let this past in #36

Reply:
Changed!